scripts/compile.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Configure to exit script as soon as a command fails.
+set -o errexit
+
+SOLC_05_DIR=solc-0.5
+
+# Delete any previous build artifacts
+rm -rf build/
+
+# Create a subproject where 0.5.x compilation will take place
+mkdir -p $SOLC_05_DIR
```

Comment:
You should always use double quotes around variables. In this case it's okay but it's a good general rule to follow.